src/crimson/osd/osd_operations/client_request.h

```diff
@@ -78,7 +78,7 @@ class ClientRequest final : public OperationT<ClientRequest> {
   PGPipeline &pp(PG &pg);
 
   OpSequencer& sequencer;
-  const uint64_t prev_op_id;
+  uint64_t prev_op_id = 0xFFFFFFFFFFFFFFFF;
```

Comment:
i think we can use `0` as a sentry value. see https://github.com/ceph/ceph/pull/39598/commits/2483a486b85621cb7a1ef19eeeac4dec7d997deb

Reply:
Ah, yes, correcting it:-)